src/Tomcat/TomcatController.ts

```diff
@@ -176,12 +182,28 @@ export class TomcatController {
         }
     }
 
+    public async buildAndDebugWar(debug:boolean,server?:TomcatServer): Promise<void>{
+        if (!server) {
+            server = await this.selectServer(true);
+        }
+
+        let warFile: string = workspace.getConfiguration("tomcat").get<string>("build.warFile");
+        warFile = warFile.split(".")[0];
```

Comment:
User define the war package need to build and deploy/debug in the settings? could you give an example of the settings and explain a little bit about the scenario?